src/webgpu/util/conversion.ts

```diff
@@ -576,6 +580,10 @@ export class VectorType {
   public toString(): string {
     return `vec${this.width}<${this.elementType}>`;
   }
+
+  public getSize(): number {
+    return this.elementType.size * this.width;
+  }
```

Comment:
nit: I think if you make this `public get size(): number` then it will be compatible with the `readonly size: number` defined in ScalarType and you can use `.size` instead of `.getSize()`